### ISSUE TITLE
Accessibility test results & recommendations

### ISSUE THREAD

**Issue by GBKS** [Blockstream/green_ios#43]
### Description

Did some quick accessibility testing on iOS using Voice Control and VoiceOver, and stumbled across a few issues:
- The account stack on the home screen is not accessible
- Recovery phrase verification is problematic
- Various missing labels on buttons and QR codes, and some mixed up selection ordering (these are super easy to fix)

Overall, the app gets a ton of things right. Fixing these other issues would make it even better.

#### Version

3.8.4

### Steps to reproduce

My test workflow is shown in [this video](https://www.youtube.com/watch?v=n9FvI5SfpoU&t=4s) (for a different app, but the workflow is the same). This was a pretty quick and casual test, I did not thoroughly go over every single screen, and also did not test very accessibility feature. I also did not include every single instance of missing labels I came across. 

### Expected behaviour

The app can be navigated without seeing what is on the screen.

### Actual behaviour

See the screenshots below for notes.

### Screenshots

[Figma file](https://www.figma.com/file/vWxA4SRJq7RdXWiKrB032y/Bitcoin-app-accessibility-tests?node-id=139%3A2433) with screenshots and annotations.

![Blockstream Green iOS accessibility test summary 220708](https://user-images.githubusercontent.com/695901/177984111-44afb2f7-e676-47e0-86e2-ef97a1ae189b.png)

#### Device or machine

iPhone 11

#### Additional info

This is part of an accessibility testing effort I am part of in the [Bitcoin Design](https://bitcoin.design) community. We gathered info about accessibility in bitcoin applications [here](https://bitcoin.design/guide/designing-products/accessibility/), and are testing our way through various wallets. Reach out if you'd like to chat more about this.

**Comment by alessandro-saglimbeni** [comment]
Than you very much for the thorough feedback @GBKS, we're going to make some UI changes shortly, so it's a good time to take these accessibility issues into account as well!

**Comment by GBKS** [comment]
Following up, I did a very quick test with the latest TestFlight version and see a lot of missing labels. Hope you guys & gals can get around to addressing these. Adding a label really just takes 30 seconds to do, but can make all the difference for a low-vision users.

![image](https://user-images.githubusercontent.com/695901/227864971-09157779-4ae9-469a-8662-8b286982d73a.png)

**Comment by GBKS** [comment]
Closing this issue as it has not seen activity in 3 years.
